backend/Utils/EmailUtils.ts

```diff
@@ -0,0 +1,56 @@
+import google from "googleapis";
+import path from "path";
+import { encode } from "js-base64";
+import dotenv from "dotenv";
+
+dotenv.config();
+
+async function makeEmail({ to, from, subject }) {
+  const message = `This is a test email for reset password.`;
+
+  // the white space is important, string template is space sensitive
```

Comment:
Nitpicky, but "string template" isn't the precisely correct term - it is "template literal." However, most people say "template string" rather than "template literal"

```suggestion
  // the white space is important, template literal is space sensitive
```

See https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals

Reply:
Thank you for this! I learned it as template literal so this entirely makes sense now.